primitives/core/src/testing.rs

```diff
@@ -22,11 +22,15 @@ use crate::crypto::KeyTypeId;
 use crate::{
 	crypto::{Pair, Public, CryptoTypePublicPair},
 	ed25519, sr25519, ecdsa,
-	traits::Error,
+	traits::{CryptoStorePtr, Error, SyncCryptoStore, SyncCryptoStorePtr},
 	vrf::{VRFTranscriptData, VRFSignature, make_transcript},
 };
 #[cfg(feature = "std")]
-use std::collections::HashSet;
+use std::{collections::{HashMap, HashSet}, sync::Arc};
+#[cfg(feature = "std")]
+use parking_lot::RwLock;
+
+use async_trait::async_trait;
```

Comment:
If I read this right, it is only ever used in `std`, right? Let's remove that useless warning for non-std-cases then...
```suggestion

#[cfg(feature = "std")]
use async_trait::async_trait;
```